tasks/s3roundtrip.py

```diff
@@ -28,13 +28,17 @@ def download(ctx, config):
     The context passed in should be identical to the context
     passed in to the main task.
     """
-    assert isinstance(config, list)
+    assert isinstance(config, dict)
```

Comment:
Yes. I changed that because I changed the downloads call to be a dict so that the new force-branch parameter can be used.  The only uses of the config parameter inside this function is in the code that I modified and in the finally code that I just checked.  I was afraid that this may break things but it looks to be correct.